frontend/src/App.js

```diff
@@ -1,10 +1,95 @@
 import './App.css';
 import Page from './components/Page/Page';
+import { useEffect, useState } from 'react';
```

Comment:
I generally try to sort imports. Usually imports from packages (e.g. React) up top and component imports below with maybe a line's worth of space separating them. This helps improve readability.